hudi-cli/src/test/java/org/apache/hudi/cli/commands/TestLockAuditingCommand.java

```diff
@@ -56,6 +61,76 @@ public class TestLockAuditingCommand extends CLIFunctionalTestHarness {
 
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
 
+  /**
+   * Represents a single audit record (JSON line in a .jsonl file)
+   */
+  static class AuditRecord {
```

Comment:
Should this be used by `StorageLockProviderAuditService` and `LockAuditingCommand`?

Reply:
I feel that the code is becoming over-engineered. I want to avoid modifying the core path too much. I will keep the generic map serialization for now. If you feel strongly enough that we should standardize on a specific object that's fine, but I want to be very flexible, in case we want to write more data later.